File: Components/WinForms/VB/PrintingTutorials/Tutorial1/LessonPreviewIntro.vb
Imports System
Imports System.ComponentModel
Imports DevExpress.XtraPrinting
Imports System.Windows.Forms
Imports System.Reflection
Imports DevExpress.Tutorials
Imports DevExpress.DXperience.Demos

Namespace XtraPrintingDemos.Tutorial1
    Public Class LessonPreviewIntro
        Inherits ModuleBase

        Private components As Container
        Private featuresRtf As RichTextBox

        Public Sub New()
            InitializeComponent()
            Dim resourceStream As System.IO.Stream = System.Reflection.Assembly.GetAssembly(GetType(LessonPreviewIntro)).GetManifestResourceStream("PrintingTutorials.BaseForms.PrintingAbout.rtf")
            If resourceStream IsNot Nothing Then
                featuresRtf.LoadFile(resourceStream, RichTextBoxStreamType.RichText)
            End If
        End Sub
        Private Sub InitializeComponent()
            components = New Container()
            featuresRtf = New RichTextBox()
            featuresRtf.Name = "featuresRtf"
            featuresRtf.Dock = DockStyle.Fill

            Controls.Add(featuresRtf)
        End Sub
    End Class
End Namespace

File: Components/WinForms/CS/PrintingTutorials/Tutorial1/LessonPreviewIntro.cs
using System;
using System.ComponentModel;
using DevExpress.XtraPrinting;
using System.Windows.Forms;
using System.Reflection;
using DevExpress.Tutorials;
using DevExpress.DXperience.Demos;

namespace XtraPrintingDemos.Tutorial1 {
    public class LessonPreviewIntro : ModuleBase {
        Container components;
        RichTextBox featuresRtf;

        public LessonPreviewIntro() {
            InitializeComponent();
            System.IO.Stream resourceStream = Assembly.GetAssembly(typeof(LessonPreviewIntro)).GetManifestResourceStream("PrintingTutorials.BaseForms.PrintingAbout.rtf");
            if(resourceStream != null)
                featuresRtf.LoadFile(resourceStream, RichTextBoxStreamType.RichText);
        }
        private void InitializeComponent() {
            components = new Container();
            featuresRtf = new RichTextBox();
            featuresRtf.Name = "featuresRtf";
            featuresRtf.Dock = DockStyle.Fill;

            Controls.Add(featuresRtf);
        }
    }
}
